app/src/components/UserAddress.vue

```diff
@@ -20,6 +30,21 @@ export default {
       return `https://avatars.dicebear.com/api/jdenticon/${this.address}.svg`;
     },
   },
+  methods: {
+    copyAddress() {
+      const clipboardData =
+        event.clipboardData ||
+        window.clipboardData ||
+        event.originalEvent?.clipboardData ||
+        navigator.clipboard;
+
+      clipboardData.writeText(this.address);
```

Comment:
We need a more reliable approach. You can also check https://www.npmjs.com/package/v-clipboard